draft-ietf-quic-http.md

```diff
@@ -1916,19 +1916,28 @@ in {{frame-reserved}} and {{stream-grease}}.  These methods of padding produce
 different results in terms of the granularity of padding, how padding is
 arranged in relation to the information that is being protected, whether padding
 is applied in the case of packet loss, and how an implementation might control
-padding.  Redundant padding could even be counterproductive.
+padding.
+
+Reserved stream types can be used to give the appearance of sending traffic even
+when the connection is idle.  Because HTTP traffic often occurs in bursts,
+apparent traffic can be used to obscure the timing or duration of such bursts,
+even to the point of appearing to send a constant stream of data.  However, as
+such traffic is still flow controlled by the receiver, a failure to promptly
+drain such streams and provide additional flow control credit can limit the
+sender's ability to send real traffic.
 
 To mitigate attacks that rely on compression, disabling or limiting compression
 might be preferable to padding as a countermeasure.
 
 Use of padding can result in less protection than might seem immediately
```

Comment:
This section starts talking about using H2 PADDING to make a connection more resistant to traffic analysis and ends discussing how padding doesn't really help that much.  Is there a use of PADDING that is recommended?

Reply:
I would frame it that padding is a tool that can be used to mitigate certain attacks, but it's not a perfect tool.  It's useful for its job, but you need to understand its drawbacks when designing a padding scheme.

Reply:
Makes sense, thanks.